partiql-ast/src/main/java/org/partiql/ast/v1/ProjectItem.java

```diff
@@ -0,0 +1,81 @@
+package org.partiql.ast.v1;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * TODO docs, equals, hashcode
+ */
+public abstract class ProjectItem extends AstNode {
```

Comment:
SelectItem.Expr
SelectItem.Star

Reply:
ack -- applied this and the other `SELECT` node renaming.